repair_tests/repair_test.py

```diff
@@ -1195,6 +1195,40 @@ def run_repair():
         else:
             node1.nodetool('repair keyspace1 standard1 -inc -par')
 
+    @since('3.0')
+    def repair_one_node_cluster_test(self):
+        self._repair_abort_test(nodes=1, rf=2)
+
+    @since('3.0')
+    def repair_one_node_in_local_dc_test(self):
+        self._repair_abort_test(options=['--in-local-dc'], nodes=[1, 1], rf={'dc1': 1, 'dc2': 1}, no_common_range=True)
+
+    def _repair_abort_test(self, options=[], nodes=1, rf=1, no_common_range=False):
+        """
+        @jira_ticket DB-1511
```

Comment:
We should not mention internal (DB) Datastax tickets in the changes that are going to be ported to the OSS.

Reply:
Good point! I was blind.